data/sea/30-array.h

```diff
@@ -315,3 +315,10 @@ MAKE_ARRAY(ARRAY(itime))
 MAKE_ARRAY(ARRAY(istring))
 MAKE_ARRAY(ARRAY(iunit))
 #endif
+
+
+typedef void* iany_t;
```

Comment:
an `Object`! It's top, it's everything except the things that matters (chars, ints) :D